### PR TITLE
Docs on how to get started running a log cluster.

### PR DESCRIPTION
**Pull request by AlCutter**

### CODE REVIEW

File: docs/ClusterHowto.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+       libssl-dev \
+       cmake \
+       google-mock \
+       protobuf-compiler \
+       libprotobuf-dev \
+       libgflags-dev \
+       libgoogle-glog-dev \
+       libsqlite3-dev \
+       libjson-c-dev \
+       libevent-2.0-5 \
```
Comment (pphaneuf) [inline docs/ClusterHowto.md]: Remove this one?

File: docs/ClusterHowto.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+
+## Prerequisites
+You should have:
+* Installed the dependencies listed in the top-level [README.md](https://github.com/google/certificate-transparency/README.md) file,
+* Installed the Docker utilities (and started the docker daemon if it's not already running)
+* Signed up for a [Google Cloud](https://cloud.google.com) account _[only required if you're intending to use GCE, of course]_
+
+### Dependencies for Debian-based distros
+Assuming you're happy to use the stock versions of dependencies, Debian-based distros (including Ubuntoo etc.) have many of them pre-packaged which will make your life easier, you can install them using the following command:
```
Comment (phad) [inline docs/ClusterHowto.md]: Ubuntoo -> Ubuntu

File: docs/ClusterHowto.md (hunk 1 above)
Comment (AlCutter) [inline docs/ClusterHowto.md]: Dinged

File: docs/ClusterHowto.md (hunk 2 above)
Comment (AlCutter) [inline docs/ClusterHowto.md]: Whoops, thanks - done.

**Comment by pphaneuf** [comment]
LGTM

**Pull request merged**